scanners/httpobs_scanner.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import time
+
+
+class HTTPObservatoryScanner():
+    ## TODO: Re-write this with the httpobs python module
+    def __init__(self, sleep_interval=1):
+        self.session = requests.Session()
+        self.sleep_interval = sleep_interval
```

Comment:
should  this be named "poll_interval" to be more straight forward?

Reply:
Will change.